boards/nrf52840-dongle/Makefile.include

```diff
@@ -0,0 +1,38 @@
+# This board uses the vendor's serial bootloader
+
+PROGRAMMER ?= nrfutil
+
+# We're not using the softdevice actually (nor use JLink as in
+# nrf52/Makefile.include), but the linker script should be a good starting
+# point for using the nRF bootloader and not breaking the soft device.
+
+LINKER_SCRIPT ?= $(BOARDSDIR)/nrf52840-dongle/ldscripts/application.ld
+
+# This is under active exploration, and should probably go into its one
+# makefiles/tools file
+
+ifeq (nrfutil,$(PROGRAMMER))
+
+FLASHFILE = $(HEXFILE).zip
```

Comment:
Why not make this `FLASHDEPS`?

```suggestion
FLASHFILE = $(HEXFILE)
FLASHDEPS = $(HEXFILE).zip
```

This way one is able to build without having `nrfutil` installed.

Reply:
I was unaware FLASHDEPS exist -- good point, doing that in 1ca582419ce6b37ec77cd7dce91708a82ed9a711. That should solve the no-chance-on-Murdock problem.

Murdock builds will still fail where USB gets enabled -- #13148 not being in yet means that examples need manual `CFLAGS += -DCONFIG_USB_PID=0x7D00 -DCONFIG_USB_VID=0x1209` additions, so I'm not sure it makes sense turning it on again yet.

Reply:
Then... Let's make this PR dependent on #13148?

Reply:
Aehm yes, how do I do that? (I'd know in Gerrit but don't know in GitHub).

Reply:
If you want to make it specific, rebase it to that PR locally (how to check the PR locally to your machine is documented below where it says "[…] or view command line instructions". But I do it logically, by just updating the OP.